test/tests/commonjs-requires-in-comment.js

```diff
@@ -0,0 +1,15 @@
+"format cjs";
```

Comment:
true, the penny has just dropped that the existing commonjs-requires.js test file gets executed by the loader, so I just moved the test into there as you originally suggested. Thanks.